src/Equinor.ProCoSys.IPO.Query/GetComments/GetCommentsQueryHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.PersonAggregate;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.GetComments
+{
+    public class GetCommentsQueryHandler : IRequestHandler<GetCommentsQuery, Result<List<CommentDto>>>
+    {
+        private readonly IReadOnlyContext _context;
+
+        public GetCommentsQueryHandler(IReadOnlyContext context) 
+            => _context = context;
+
+        public async Task<Result<List<CommentDto>>> Handle(
+            GetCommentsQuery request,
+            CancellationToken token)
+        {
+            var invitation = await
+                (from inv in _context.QuerySet<Invitation>()
+                        .Include(i => i.Comments)
+                        .Where(i => i.Id == request.InvitationId)
+                 select inv).SingleOrDefaultAsync(token);
+
+            if (invitation == null)
+            {
+                return new NotFoundResult<List<CommentDto>>($"Entity with ID {request.InvitationId} not found");
```

Comment:
return new NotFoundResult<InvitationDto>(Strings.EntityNotFound(nameof(Invitation), request.InvitationId));
Search for NotFoundResult in this project and you'll see other statements as this which should use Strings.EntityNotFound